client/dashboard/sites/site-fields/index.tsx

```diff
@@ -224,6 +238,50 @@ export function Preview( { site }: { site: Site } ) {
 	);
 }
 
+export function Preview__ES( { site }: { site: DashboardSiteListSite } ) {
+	const [ resizeListener, { width } ] = useResizeObserver();
+
+	// If the site is a private A8C site, X-Frame-Options is set to same
+	// origin.
+	const iframeDisabled = site.deleted || ( site.is_a8c && site.private );
+	return (
+		<Link
+			to={ getSiteManagementUrl__ES( site ) }
+			disabled={ site.deleted }
+			style={ {
+				display: 'block',
+				height: '100%',
+				width: '100%',
+				borderRadius: 'inherit',
+				overflow: 'hidden',
+			} }
+		>
+			{ resizeListener }
+			{ iframeDisabled && (
+				<div
+					style={ {
+						fontSize: '24px',
+						display: 'flex',
+						alignItems: 'center',
+						justifyContent: 'center',
+						height: '100%',
+					} }
+				>
+					<SiteIconRenderer
```

Comment:
We need to switch to `SiteIconRenderer`, so I create `Preview__ES` instead of `PreviewRenderer`